frontend/app_flowy/assets/translations/en.json

```diff
@@ -163,8 +163,14 @@
         "light": "Light Mode",
         "dark": "Dark Mode",
         "system": "Adapt to System"
+      },
+      "themeType": {
```

Comment:
Can we use appTheme or theme? ThemeType seems to just appear from nowhere

Reply:
Okay sure

Reply:
What do u want it to be, appTheme or just theme. I prefer theme.

Reply:
theme sounds good to me